fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/api/content/registry/v1/TillableBlockRegistry.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.content.registry.v1;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+
+import net.fabricmc.fabric.api.content.registry.v1.util.ContentRegistry;
+import net.fabricmc.fabric.impl.content.registry.ToolBlockTransformationRegistries;
+
+/**
+ * Registry of Blocks that when tilled turn into a specific BlockState.
+ */
+public interface TillableBlockRegistry extends ContentRegistry<Block, BlockState> {
```

Comment:
imo the extends clause is useless and confusing. This interface is effectively a constant holder.

Reply:
Not at my pc right now but I believe you're right, the tillable registry doesn't add any extra methods so it shouldn't need this